src/pages/kb/user-guide/querying/writing-queries.md

```diff
@@ -8,65 +8,70 @@ toc: true
 order: 1
 ---
 
-To start working on a new query, click "Create" in the nav then select "Query".
+To make new query, click `Create` in the navbar then select `Query`.
 
 ![](/assets/images/docs/gifs/queries/add_new_query.gif)
 
-### Publish/Unpublish
-
-By default each query starts in a draft state (Unpublished), which means that:
-
-- Only the user who created this query can see it in the "All Queries" list or in search results.
-- You can't add visualizations from an unpublished query to dashboards or use it in alerts.
+### Query Syntax
 
-To publish a query you can simply give it a name or click the "Publish"
-button. It's also possible to unpublish a published query by clicking on the
-"Unpublish" button in the query menu.
+In most cases we use the query language native to the data source. In some cases there are differences or additions, which are documented on the [Querying Data Sources]({% link _kb/data-sources/querying %}) page.
 
-### Query Syntax
+### Keyboard Shorcuts
 
-In most cases we use the data sources native query language. In some cases
-there are differences or additions, which are documented in the [Querying Data
-Sources category]({% link _kb/data-sources/querying %}).
+- Execute query: `Ctrl`/`Cmd` + `Enter`
+- Save query: `Ctrl`/`Cmd` + `S`
+- Toggle Auto Complete: `Ctrl` \+ `Space`
```

Comment:
Is the `\+` on purpose there?

Reply:
It wasn't on purpose. But it also renders properly. I'll remove it though.